### DISCUSSION TITLE
Create a scene with SolidWorks

### DISCUSSION THREAD

**Discussion by Pierre-Moulin** [Ifsttar/I-Simpa#328] [Ideas]
Hello everyone,
Thanks to your help, I finally clear importing a scene made on SolidWorks :) 
So here it is a tutorial of my method  : 

1) Create your room :
Create 3 sketches with using the 3 planes (front/top/right) and use Insert<Surface<Planar for having 3 surfaces
Create the 3 opposite walls with using Surface offset 

![image](https://user-images.githubusercontent.com/106664450/186864920-a0998b38-9a4e-4a2a-9519-e92cd75cf83e.png)

2) Create your machines : 
**You have to create your machines, with volumes, above the floor (I use 1cm gap)** otherwise you will have interactions problems on I SIMPA.
Use the function Surface<Offset and select all the surfaces of the machines with using an offset of 0mm
Then delete the Solid bodies in the tree 

3) Save : 
Save your assemblage as a part
Then save the part in PLY

![image](https://user-images.githubusercontent.com/106664450/186867494-1c2b28f9-9551-40a9-aba3-09bdc3a0fa8a.png)

**Take care of the units and the Origin**  (not as me ... ) ;)

**Comment by Picaut** [comment]
Thank you very much for this tutorial. This will benefit for the community.
J. Picaut
